### DISCUSSION TITLE
Galaxy Brain badge not found even after answering more than 2 discussions.

### DISCUSSION THREAD

**Discussion by ELHart05** [community/community#145787] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The requirement for the Galaxy Brain badge is **to answer 2 discussions**. But I´ve already done so and no badge was given to me.

Here are the discussions that marked my answer:
- https://github.com/orgs/community/discussions/145772
- https://github.com/orgs/community/discussions/145557
- https://github.com/orgs/community/discussions/145662

Is there another requirement to get the badge?

**Comment by rehanawan** [comment]
You will get it when your answers are marked as valid answers. Not just answering on discussions get you the badge. the owner of the thread has to mark your answer.

**Reply by ELHart05** [reply]
No!, the discussions marked my answer as correct (I have also updated my question with the discussions that marked my answers)

**Reply by rehanawan** [reply]
Update: this badge is no longer available. 
People were posting very nonsense content to get this badge to github has decided to revoke this acheivement.

[comment on other similar discussion](https://github.com/orgs/community/discussions/48448#discussioncomment-8469056)
[https://github.com/orgs/community/discussions/106536](https://github.com/orgs/community/discussions/106536)
